libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -233,6 +240,46 @@ describe('ModalHelper', () => {
           });
         });
 
+        describe(`drawer can interact with background`, () => {
+          beforeEach(async () => {
+            await openDrawer('Drawer On Presenting Element', undefined, 'medium', true);
+          });
+
+          afterEach(async () => {
+            await overlay.dismiss();
+          });
```

Comment:
Consider inlining in each `it` block (as one of the tests already dismisses the overlay)

Reply:
Fixed in next commit